package.json

```diff
@@ -22,12 +22,15 @@
 		"clean": "rimraf -g **/node_modules **/tests/**/package-lock.json **/examples/**/package-lock.json **/dist **/coverage packages/fuzzer/build packages/fuzzer/prebuilds",
 		"compile:watch": "tsc -b tsconfig.build.json --incremental --pretty --watch",
 		"test": "run-script-os",
-		"test:jest": "jest && npm run test --ws --if-present",
-		"test:coverage": "jest --coverage",
 		"test:default": "npm run test:jest",
-		"test:linux:darwin": "npm run test:jest && cd examples && sh ../scripts/run_all.sh dryRun && cd ../tests && sh ../scripts/run_all.sh fuzz",
-		"test:win32": "npm run test:jest && cd examples && ..\\scripts\\run_all.bat dryRun && cd ..\\tests && ..\\scripts\\run_all.bat fuzz",
-		"test:watch": "jest --watch",
+		"test:linux:darwin": "npm run test:jest && cd tests && sh ../scripts/run_all.sh fuzz",
+		"test:win32": "npm run test:jest && cd tests && ..\\scripts\\run_all.bat fuzz",
+		"test:jest": "jest && npm run test --ws --if-present",
+		"test:jest:coverage": "jest --coverage",
+		"test:jest:watch": "jest --watch",
+		"example": "run-script-os",
```

Comment:
Weren't the examples already covered under `npm run test`?

Reply:
Yes, I extracted them out to execute only the examples with a different node.js version, but reverted that part back. Still, this should reduce a few minutes from the overall runtime as es examples are now executed in parallel.